12_listcomprehension_lab/01.py

```diff
@@ -1,8 +1,5 @@
-"""
-Use range() and list comprehension to get
-the list of all lowercase english letters
-Hint: look for chr() and ord()
-"""
-
-
-
+#List of all lower case letters
+
+#Generate a list of all lower letters
+lowerletters = [chr(index) for index in range(1,256) if index >= ord('a') and index <= ord('z')]
+print " ".join(lowerletters)

```

Comment:
It would be better to have the range from ord('a') till ord('z') and then there is no need for the 'if' statement.